packages/stories/.storybook/tsconfig.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
Was it useless?

Reply:
yes, there was no typescript files in that folder